src/pages/createPost.tsx

```diff
@@ -1,172 +1,184 @@
 import { useEffect, useState, useContext, ChangeEvent } from "react";
 import { Dayjs } from "dayjs";
-
 import { useRouter } from "next/router";
 
 import { userContext } from "supabase/user_context";
 import { useSupabaseClient } from "@supabase/auth-helpers-react";
 import { Database } from "supabase/db_types";
-
-import {
-  Grid,
-  Typography,
-  Button,
-  FormHelperText,
-  Stack,
-  Box,
-} from "@mui/material";
+import { Typography, Stack, Box, Card, IconButton } from "@mui/material";
 
 import Loading from "@/components/public/Loading";
 import Navbar from "@/components/public/Navbar";
-import CommonTextField from "@/components/public/CommonTextField";
-import CommonDateTimePicker from "@/components/public/CommonDateTimePicker";
-import Tags from "@/components/createPost/Tags";
-import GoogleMap from "@/components/createPost/searchMaps";
-import PictureList from "@/components/createPost/pictureList";
+import CommonDateTimePicker from "@/components/post/CommonDateTimePicker";
+import Tags from "@/components/post/SelectTags";
+import PictureList from "@/components/post/SelectImages";
+import { validateDate, validateDateWithInterval, validateTextField } from "@/utilities/validation";
 
+import { validation } from "@/types/Validation";
 import { Tag } from "@/types/Tag";
-import { PagePaths } from "enum/pages";
-import { CHAR_LIMIT } from "enum/inputLimit";
+import { PAGE_PATHS } from "enum/PAGES";
+import { CHAR_LIMIT } from "enum/INPUT_LIMIT";
 
-import {
-  validateDate,
-  validateDateWithInterval,
-  validateTextField,
-} from "@/utilities/validation";
-import { validation } from "@/types/Validation";
 import { GetAllTags } from "@/services/Tags";
 import { CreatePost } from "@/services/Posts";
 import { PostInfo } from "@/types/Post";
+import { ICONS } from "enum/ICONS";
+import DescriptionTextField from "@/components/public/DescriptionTextField";
+import LocationTextField from "@/components/post/LocationTextField";
+import CommonButton from "@/components/public/CommonButton";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import TitleTextField from "@/components/post/TitleTextField";
 
-//style
-const createPostLayout = {
-  width: "35vw",
-  minWidth: "300px",
-  margin: "2vh 0 0 0",
+type CreatePostInput = {
+  title: string;
+  location: string;
+  startDate: Dayjs | null;
+  endDate: Dayjs | null;
+  tags: Tag[];
+  description: string;
+  images: string[];
+};
+
+type CreatePostSubmit = {
+  title: boolean;
+  location: boolean;
+  date: boolean;
+  tags: boolean;
+  description: boolean;
+  images: boolean;
+};
+
+const CreatePostStyle = {
+  TextField: {
+    width: "28vw",
+    minWidth: "250px",
+  },
+  Card: {
+    width: "30vw",
+    minWidth: "300px",
+    height: "70vh",
+    paddingTop: "2vh",
+  },
```

Comment:
`Card` is a little too short ka
<img width="482" alt="image" src="https://user-images.githubusercontent.com/62459518/226838000-f11010ef-57bf-425f-9708-4cf9ac2b095d.png">

Reply:
Solved

Reply:
I still see it this way na + please add space below the `Create Post` button just like in `src/pages/post/[post_id].tsx` ka

Reply:
Just make sure u didt zoom in right?

Reply:
yes

Reply:
Please share your display size and web browser because there is no problem on my small edge's screen 
![image](https://user-images.githubusercontent.com/69895436/226856425-4657b672-0ad8-457f-b71e-ccb1b6024143.png)

Reply:
In edit post it works just fine, did you forget to commit or something because the pic you attach is from edit Post (I guess) but I dont see the change in this comment which is create post.

Reply:
Oh dear, I correct wrong file. My bad.
Solved

Reply:
could you add space below `Create Post` button too? just like in `src/pages/post/[post_id].tsx` ka

Reply:
Added